js/scorm/wrapper.js

```diff
@@ -116,13 +121,17 @@ class ScormWrapper {
     this.lmsConnected = this.scorm.init();
 
     if (!this.lmsConnected) {
-      this.handleError(new ScormError(CLIENT_COULD_NOT_CONNECT));
+      this.handleInitializeError();
       return this.lmsConnected;
     }
 
+    if (this.connectionTest._isEnabled) {
+      Adapt.on('tracking:connectionError', () => this.handleConnectionError(false));
```

Comment:
This logic create a really weird loop between the `Connection` class and the `handleConnectionError` function and the `tracking:connectionError` event.
It would be preferable to call the `tracking:connectionError` from one place only.